geti_sdk/demos/demo_projects/coco_demos.py

```diff
@@ -382,6 +383,9 @@ def ensure_trained_example_project(
                 number_of_images_to_annotate=45,
                 enable_auto_train=True,
             )
+            # Should wait for some time for the job to appear as scheduled before checking if
+            # the project is trained. Auto training is triggered after around 5 seconds.
+            time.sleep(10)
```

Comment:
1) Can this be reduced? 10 seconds can feel a lot. 
2) Please add a log/print message here. 
 

Otherwise, it might look like a creation of the project is stuck

Reply:
Debounce time is around 5 seconds but this is not fixed. 10 feels the safest option here.